components/template/Templates.vue

```diff
@@ -1,47 +1,79 @@
 <template>
   <div>
-    <Hero @searchInput="handleSearch" />
-    <div class="template_container d-flex">
+    <div class="container template-container d-flex">
       <div class="left-sidebar">
         <TemplateCategories
           :activeCategory="activeCategory"
           :templateCategories="templateCategories"
         />
       </div>
-      <div class="w-100">
-        <h2 class="heading">
+      <div class="content-wrapper w-100">
+        <div
+          v-if="activeCategory"
+          class="breadcrumb d-flex align-items-center gap-2"
+        >
+          <NuxtLink to="/templates/" class="breadcrumb-text"
+            >All Templates</NuxtLink
+          >
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            width="16"
+            height="16"
+            viewBox="0 0 16 16"
+            fill="none"
+          >
+            <path
+              d="M6 12L10 8L6 4"
+              stroke="#6434D0"
+              stroke-width="1.33333"
+              stroke-linecap="round"
+              stroke-linejoin="round"
+            />
+          </svg>
```

Comment:
Can be an icon component

Reply:
it is a smaller svg, so I used it directly. Okay, I will turn this into a component.